src/longopt/gmtinfo_inc.h

```diff
@@ -24,33 +24,25 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 	/* separator, short_option, long_option,
 	          short_directives,    long_directives,
 	          short_modifiers,     long_modifiers */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'E', "",
-	          "",                  "",
+	{ 0, 'A', "report",
+	          "a,t,s",             "all,per_table,per_segment",
 	          "",                  "" },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'I', "",
-	          "",                  "",
+	{ 0, 'C', "columns",        "", "", "", "" },
+	{ 0, 'D', "center",            "", "", "", "" },
+	{ 0, 'E', "get_record",
+	          "l,L,h,H",           "min,minabs,max,maxabs",
 	          "",                  "" },
-	{ 0, 'L', "",
-	          "",                  "",
+	{ 0, 'F', "counts",
+	          "i,d,t",             "totals,segments,segments_reset",
 	          "",                  "" },
-	{ 0, 'S', "",
+	{ 0, 'I', "inc|increment|spacing",
+	          "b,e,f,p,s",         "box,exact,fft,override,surface",
+	          "e,r,R",             "extend_box,adjust,extend_region" },
+	{ 0, 'L', "common_limits",     "", "", "", "" },
+	{ 0, 'S', "errorbar_space|forerrorbars", "", "", "", "" },
```

Comment:
In Julia -S is called `for_error_bars`, not `forerrorbars`. For me `errorbar_space` is not intuitive.

Reply:
Have you ever used **-S**? I think this was something Stephan Erickson (?) got us to put in decades ago. Does it really serve a purpose? Better to quietly remove but let it live under the hood.